microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Crash analyzer tool
+description: How to use the Crash analyzer tool in DevTools to analyze and diagnose crashes of your production web app.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 05/14/2024
+---
+# Crash analyzer tool
+
+Use the **Crash analyzer** tool to analyze and quickly diagnose crashes of your production web app.  In the **Crash analyzer** tool, you can input a JavaScript stack trace, such as for non-fatal JavaScript exceptions, and then have your source maps applied to the stack trace so that you can debug faster.  The **Crash analyzer** tool enables you to take a minified JavaScript stack trace and rapidly work backwards to determine what lines of code caused the error.
+
+![The Crash analyzer tool](./index-images/crash-analyzer.png)
+<!-- todo: Create new demo page & use it in this article.
+Or, use page/file:
+https://devblogs.microsoft.com/oldnewthing/
+devtools://devtools/bundled/panels/elements/elements.js
+devtools://devtools/bundled/ui/components/tree_outline/tree_outline.js
+
+show stacktrace content in DevTools
+-->
+
+When a web app crashes or hangs unexpectedly, this can result in data loss and a poor user experience.  The **Crash analyzer** tool helps you identify the cause of these crashes.  Use the **Crash analyzer** tool to analyze the memory state of your web app at the time of a crash.  You can access The **Crash analyzer** tool from the **Memory** tool, which displays a list of recent crashes<!-- todo: true? --> that occurred in your web app.
+
+As a web developer, you collect stack traces from customers in production, and then use the **Crash analyzer** tool to "beautify" the stack traces in DevTools.  DevTools only does the analysis of the stack traces, not the collecting of stack traces.
+
+As an alternative approach, you can collect local stack traces by triggering a crash dump manually by clicking the **Take snapshot** button in the **Memory** tool.<!-- todo: true? -->
```

Comment:
Can you collect local stack traces by triggering a crash dump manually by clicking the **Take snapshot** button in the **Memory** tool?  per blog

Reply:
deleted sentence